app/integrations/sequencescape/api/v2/base.rb

```diff
@@ -3,6 +3,17 @@
 class Sequencescape::Api::V2::Base < JsonApiClient::Resource # rubocop:todo Style/Documentation
   class_attribute :plate, :tube, :tube_rack
 
+  # Implement a find method that raises a ResourceNotFound error if no record is found.
+  # Calls the standard find method, and raises if the result is nil.
+  # Should this be rolled into the JsonApiClient gem?
+  # @raise [JsonApiClient::Errors::NotFound] if no record is found
+  def self.find!(*)
```

Comment:
Can you do this via https://github.com/JsonApiClient/json_api_client/blob/0499153d47dfb21ff72576d5aff98b1fac035aeb/README.md?plain=1#L684

Reply:
Ah ok, makes sense. I think the JsonApiClient::Errors::NotFound returns if the resource type doesnt exist from the api. Where as here we have no records, not no resource.